hydra-node/test/Hydra/API/ServerSpec.hs

```diff
@@ -267,9 +277,109 @@ spec = describe "ServerSpec" $
                     Right (timedOutputs :: [TimedServerOutput SimpleTx]) ->
                       seq <$> timedOutputs `shouldSatisfy` strictlyMonotonic
 
+    it "displays correctly headStatus in a Greeting message" $
+      showLogsOnFailure $ \tracer ->
+        withFreePort $ \port -> do
+          withAPIServer @SimpleTx "127.0.0.1" port alice mockPersistence tracer noop $ \Server{sendOutput} -> do
+            withClient port "/?history=no" $ \conn -> do
+              status <- waitMatch 5 conn $ \v -> v ^? key "headStatus"
+              status `shouldBe` Aeson.String "Idle"
+
+            -- TODO: do test commit outputs as well? (or just unit/property test the projection for details)
```

Comment:
There is this todo I am unsure what to do about. Seems like our `projectHeadStatus` and `projectSnapshotUtxo` are not good fit for a property test? Or maybe I just suck at this and need a helping hand.

Reply:
I think we can just drop this. It is a "test that `HeadIsOpen` corresponds to `Open`" in disguise - which is not a helpful test.

We can even think of dropping 1-2 of the other the `sendOutput` + expectations blocks in this test. In the end, this is an integration test that kind of makes sure that the server output is aggregated into the `headStatus` field.